### DISCUSSION TITLE
dua doesn't show accuarate size less than 4.00 KiB

### DISCUSSION THREAD

**Discussion by akkadaya** [Byron/dua-cli#243] [Q&A] [answered]
as the title says, is it supposed to do that? if so, which tool should I use to show accurate size beside `ls -la` command?

**Comment by Byron** [comment, marked as answer]
The size it provides is accurate as it counts the amounts of blocks a file uses, and small files use at least one block.
You are probably looking for `dua -A`.

**Reply by akkadaya** [reply]
Great, thanks mate!
